integrationExamples/gpt/permutiveRtdProvider_example.html

```diff
@@ -0,0 +1,223 @@
+                      placementId: 13144370,
+                      keywords: {
+                        inline_kvs: ['1']
+                      }
+                    }
+                  },
+                  {
+                    bidder: 'rubicon',
+                    params: {
+                      accountId: '9840',
+                      siteId: '123564',
+                      zoneId: '583584',
+                      inventory: {
+                        area: ['home']
+                      },
+                      visitor: {
+                        inline_kvs: ['1']
+                      }
+                    }
+                  },
+                  {
+                    bidder: 'ozone',
+                    params: {
+                      publisherId: 'OZONEGMG0001',
+                      siteId: '4204204209',
+                      placementId: '0420420500',
+                      customData: [
+                        {
+                          settings: {},
+                          targeting: {
+                            inline_kvs: ['1', '2', '3', '4']
+                          }
+                        }
+                      ],
+                      ozoneData: {}
+                    }
+                  }
+                ]
+              },
+              {
+                code: '/19968336/header-bid-tag-1',
+                mediaTypes: {
+                  banner: {
+                    sizes: div_2_sizes
+                  }
+                },
+                bids: [
+                  {
+                    bidder: 'appnexus',
+                    params: {
+                      placementId: 13144370
+                    }
+                  },
+                  {
+                    bidder: 'ozone',
+                    params: {
+                      publisherId: 'OZONEGMG0001',
+                      siteId: '4204204209',
+                      placementId: '0420420500'
+                    }
+                  }
+                ]
+              }
+            ];
+
+
+            var googletag = googletag || {};
+            googletag.cmd = googletag.cmd || [];
+            googletag.cmd.push(function() {
+                googletag.pubads().disableInitialLoad();
+            });
+
+            var pbjs = pbjs || {};
+            pbjs.que = pbjs.que || [];
+
+            pbjs.que.push(function() {
+              pbjs.setConfig({
+                debug: true,
+                realTimeData: {
+                  auctionDelay: 800, // maximum time for RTD modules to respond
+                  dataProviders: [
+                    {
+                      name: 'permutive',
+                      waitForIt: true,
+                      params: {
+                        acBidders: ['appnexus', 'rubicon', 'ozone'],
+                        maxSegs: 500,
+                        overwrites: {
+                          rubicon: function (bid, data, acEnabled, utils, defaultFn) {
```

Comment:
FWIW - while this is ok for Feb 10, it hoped/expected that most adapters will soon be able to start reading ortb2.user.data to get segments. When rubicon supports this, will want to update the example.

Reply:
Good shout - please see comment below